react-app/src/api/govAPI.ts

```diff
@@ -33,21 +30,18 @@ export interface TallyParams {
    * Minimum percentage of total stake needed to vote for a result to be
    *  considered valid.
    */
-  quorum: number;
+  quorum: Uint8Array;
   /** Minimum proportion of Yes votes for proposal to pass. Default value: 0.5. */
-  threshold: number;
+  threshold: Uint8Array;
   /**
    * Minimum value of Veto votes to Total votes ratio for proposal to be
    *  vetoed. Default value: 1/3.
    */
-  vetoThreshold: number;
+  vetoThreshold: Uint8Array;
```

Comment:
sor, for base unit i meant in the form of `0.xxx` and then convert it to percentage in the UI component

Reply:
also have we tried the `Decimal` type?